### PR TITLE
Added setup.py to fulfill chef requirements

### PR DESCRIPTION
**Pull request by matthewrsj**
Chef looks for setup.py when installing formsender. This adds setup.py with all the application dependencies.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    'pydns==2.3.6',
+    'redis==2.10.3',
+    'validate-email==1.3',
+    'Werkzeug==0.10.4',
+    'wheel==0.24.0'
+]
+
+setup(
+    name='formsender',
+    version='1.0.0',
```
Comment (tschuy) [inline setup.py]: For now I'd keep it at something like ``0.1.0``.

**Comment by Kennric** [comment]
This looks good, but you should add the license to the license field, since we do actually have one - Apache v2

**Comment by Kennric** [comment]
I'm happy with this. +1

**Comment by tschuy** [comment]
+1

**Pull request merged**